.pre-commit-config.yaml

```diff
@@ -12,4 +12,4 @@ repos:
   rev: v4.3.4
   hooks:
   - id: isort
-    language_version: python3.7
+    language_version: python
```

Comment:
Is it not possible to use =>3.7 in pre-commit ?

Reply:
A change in the version to `>=3.7` gives this on commit :
```bash
return code: 1
expected return code: 0
stdout:
    RuntimeError: Failed to find interpreter for Builtin discover of python_spec='python>=3.7'
```
 Managed to take a look at the documentation for `pre-commit`, specifically on [Overriding language version](https://pre-commit.com/#overriding-language-version). From what I could tell, you cannot include a `>=` on that,

My attempt at using the previous config file with `python3.7` failed. This means I had to install [pyenv](https://github.com/pyenv/pyenv#understanding-path) to manage multiple python versions, followed by [pyenv-virtualenv](https://github.com/pyenv/pyenv-virtualenv) to manage these environments. Still, this does not create a virtual environment in the current directory as I have seen the `hooks` attempt to work with a file `.venv` with the assumption, this is the default env.

Reply:
Hm yeah I see. I think we should use `python3` then, as python2 is at EOL.

Reply:
The [update](https://github.com/epadctf/Paddown/pull/30/commits/c7dcd4fd78a54be0bbb2b7d09ab04bab68e4747d) should work.